MPCAutofill/MPCAutofill/settings.py

```diff
@@ -11,27 +11,34 @@
 """
 
 import os
-
-from environ import Env
-
-env = Env()
-env.read_env()
+import environ
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-# Production-specific settings kept in local_settings.py
+# Production-specific settings kept in .env, modify the .env.dist then rename
+env = environ.Env(
+    # set casting, default value
+    DEBUG=(bool, False)
+)
+env.read_env(os.path.join(BASE_DIR, "MPCAutofill/.env"))

```

Comment:
I think I read somewhere that defining the exact location and name of the .env file was best practice but I don't remember why they said that, it could have very well been outdated information. I think it was in a digital ocean django primer or something? In any case I'm totally agnostic on this change so we can change it back if you want!